ern-local-cli/src/index.ts

```diff
@@ -115,12 +139,16 @@ export default function run() {
   kax.renderer =
     logLevel === 'trace' || logLevel === 'debug'
       ? new KaxSimpleRenderer(kaxRendererConfig)
-      : new KaxAdvancedRenderer(kaxRendererConfig)
+      : logLevel === 'off'
```

Comment:
do we really want to complicate here? let's keep default render as the option. as documented in the packages wiki

Reply:
Not sure that I get what you mean here ?
Basically if the `--json` option is set, we want to disable all kind of logs (thus setting logLevel to 'off' level). 
This includes any log done through `kax`. I don't see another simpler way (or maybe avoid the ternary and use if/else blocks instead which would make the code more readable ?)